### DISCUSSION TITLE
Can't extend the theme for Textarea

### DISCUSSION THREAD

**Discussion by sethstha** [chakra-ui/chakra-ui#5466] [Q&A]
I see Input and Textarea share the same config. While the config works fine for the Input it doesn't work Textarea.

BorderColor is not working

```
 sizes: {
    md: {
      field: {
        rounded: 'none',
      },
    },
  },
  variants: {
    flushed: {
      field: {
        borderColor: '#ccc',
      },
    },
  },
  defaultProps: {
    variant: 'flushed',
  },
```

**Comment by alexmccdev** [comment]
I had luck removing `field`:

```
 sizes: {
    md: {
      rounded: 'none',
    },
  },
  variants: {
    flushed: {
      borderColor: '#ccc',
    },
  },
  defaultProps: {
    variant: 'flushed',
  },
```

**Comment by avamolnar** [comment]
tl;dr; Check to see if you're applying your Defined Style Config to `Textarea` (and NOT Text**A**rea (which doesn't exist))

I discovered a similar issue - but it turned out to PEBKAC (Problem existed between keyboard and chair).

In my case in my theme file I'd applied the newly created slides to a component called TextArea - which doesn't exist.

Make sure you apply the styles to `Textarea`

**Reply by mjviana** [reply]
Damn you PascalCase!!
Been stuck on exactly this for the the last 1h.

Thanks a lot!
